### DISCUSSION TITLE
Connecting LibreChat to Flowise

### DISCUSSION THREAD

**Discussion by themw123** [danny-avila/LibreChat#1895] [Feature Requests & Suggestions]
Flowsie lets us build ChatBots with a graphical user interface with its underlying LangChain Framework. Flowise serves its ChatBots with an api. It would be nice to use and integrate the ChatBots of Flowise in LibreChat

**Comment by Daniel-DDV** [comment]
I agree, should not be too hard and would be awesome!

**Comment by peterclemenko** [comment]
this would indeed be awesome
